packages/common/src/index.ts

```diff
@@ -374,6 +384,26 @@ export interface Web3Instance {
 
 export { graphQlEndpoints, graphql_fetch } from "./graphql_fetch";
 
+export function roundToPassportCommunityIdMap(round: Round) {
```

Comment:
let's return here both the id and the key, so that we don't have the same check in 2 different places. 
we can return something like {communityId: ..., apiKey: ...}